opi/config/__init__.py

```diff
@@ -0,0 +1,33 @@
+import os
+import pwd
+import subprocess
+import json
+from functools import cache
```

Comment:
Have a look at the testsuite: https://github.com/openSUSE/opi/runs/2906061366?check_suite_focus=true

Reply:
I think the test should work now, but I can not run them because I can not get podman to work.